src/ui/components/AnswerOption/AnswerOption.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import AnswerOption from './AnswerOption';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+  title: 'Forms/AnswerOption',
+  component: AnswerOption,
+  tags: ['autodocs'],
+} satisfies Meta<typeof AnswerOption>;
+
+export default meta;
```

Comment:
This file exports 'meta' as the default export (export default meta;). Project conventions require named exports for modules. Note: Storybook itself requires a default export for CSF meta; this is a known exception to the rule. The real issue is that the component file currently uses a default export as well, which violates project rules and complicates imports.

**Sugerencia:**
```typescript
Keep Storybook's default export (it is required by Storybook), but change the component to a named export as suggested earlier. Alternatively, if the team insists on no default exports in stories, standardize and document an exception for Storybook meta in your contribution guidelines.
```